spec/support/macros.rb

```diff
@@ -0,0 +1,15 @@
+def set_current_user(user=nil)
+  @user = user || Fabricate(:user)
```

Comment:
Instance variable? :) This does not look good to me, will get carried to other specs? You should use local variables definitely in specs and also this should be memoized as shown below.